droidlet/dialog/map_to_dialogue_object.py

```diff
@@ -114,10 +114,13 @@ def handle_logical_form(
             return self.dialogue_objects["bot_capabilities"](memory=memory)
         elif logical_form["dialogue_type"] == "HUMAN_GIVE_COMMAND":
             low_level_interpreter_data = {}
```

Comment:
can you put a FIXME here so we can remember to excise this more cleanly in later PR?

Reply:
Yeah definitely

Reply:
can we at least have a special dict for these things, instead of putting them in "opts"?

and a BIG FIXME!!!! here